src/sentry/static/sentry/app/routes.jsx

```diff
@@ -908,6 +908,21 @@ function routes() {
               component={errorHandler(LazyLoad)}
             />
           </Route>
+          <Route
+            path="/organizations/:orgId/incidents/"
+            componentPromise={() =>
+              import(/* webpackChunkName: "OrganizationIncidentsContainer" */ './views/organizationIncidents/index')
```

Comment:
What's the reason for explicitly including `index` here? Doesn't that defeat the purpose of an index file?

Reply:
Just seems to be done like this in a few places in the routes file, I can remove if you like

Reply:
Yeah can we remove all of the `/index` imports?

Reply:
Sounds good to me